src/i18n/locales/ml.ts

```diff
@@ -0,0 +1,415 @@
+import type { LanguageMetadata, LanguageTranslation } from '../types';
+
+export const ml: LanguageTranslation = {
+    translation: {
+        menu: {
+            file: {
+                file: 'ഫയൽ',
+                new: 'പുതിയത്',
+                open: 'തുറക്കുക',
+                save: 'സംരക്ഷിക്കുക',
+                import_database: 'ഡാറ്റാബേസ് ഇറക്കുമതി ചെയ്യുക',
+                export_sql: 'SQL കയറ്റുമതി ചെയ്യുക',
+                export_as: 'ആയി കയറ്റുമതി',
+                delete_diagram: 'ഡയഗ്രം ഇല്ലാതാക്കുക',
+                exit: 'പുറത്ത്',
+            },
+            edit: {
+                edit: 'തിരുത്തുക',
+                undo: 'പഴയപടിയാക്കുക',
+                redo: 'വീണ്ടും ചെയ്യുക',
+                clear: 'വ്യക്തമായ',
+            },
+            view: {
+                view: 'കാഴ്ച',
+                show_sidebar: 'സൈഡ്‌ബാർ കാണിക്കുക',
+                hide_sidebar: 'സൈഡ്‌ബാർ മറയ്ക്കുക',
+                hide_cardinality: 'കാർഡിനാലിറ്റി മറയ്ക്കുക',
+                show_cardinality: 'കാർഡിനാലിറ്റി കാണിക്കുക',
+                zoom_on_scroll: 'സ്ക്രോളിൽ സൂം ചെയ്യുക',
+                theme: 'തീം',
+                show_dependencies: 'ആശ്രിതത്വം കാണിക്കുക',
+                hide_dependencies: 'ഡിപൻഡൻസികൾ മറയ്ക്കുക',
+            },
+            share: {
+                  share: 'പങ്കിടുക',
+                  export_diagram: 'ആരാഖം ഇറക്കുമതി ചെയ്യുക',
+                  import_diagram: 'ആരാഖം ഇറക്കുമതി ചെയ്യുക',

```

Comment:
You missed here!
update the translation of the import_digram

Reply:
@KareemAdelAwwad done!